CHANGELOG.md

```diff
@@ -1,3 +1,131 @@
+# Release 0.22.0
+
+## Breaking Changes
+Commands under `ziti edge` now reserver the `-i` flag for specifying client identity. 
+Any command line argumet which previously had a `-i` short version now only has a long version.
+
+For consistency, policy roles parameters must all be specified in long form
+
+This include the following flags:
+* ziti edge create edge-router-policy --identity-roles --edge-router-roles
+* ziti edge update edge-router-policy --identity-roles --edge-router-roles
+* ziti edge create service-policy --identity-roles --service-roles
+* ziti edge update service-policy --identity-roles --service-roles
+* ziti edge create service-edge-router-policy --service-roles --edge-router-roles
+* ziti edge update service-edge-router-policy --service-roles --edge-router-roles
+* ziti edge create posture-check mfa --ignore-legacy
+* ziti edge update posture-check mfa --ignore-legacy
+* ziti edge update authenticator updb --identity 
+* ziti egde update ca --identity-atributes (now -a)
+
+The `ziti edge` commands now store session credentials in a new location and new format. Existing sessions will be ignored.
+
+## What's New
+
+* Feature: Enhanced `ziti` CLI login functionality
+* Feature: new `ziti edge list summary` command, which shows database entity counts
+
+## Enhanced CLI Login Functionality
+### Certificate Management
+#### Working with self signed certs
```

Comment:
Title chainge of some kind. Maybe `Working With CAs` or `Validating Server Identity` or `Server Trust`.

Reply:
Fixed